packages/zoe/test/unitTests/contracts/test-callSpread.js

```diff
@@ -620,3 +601,121 @@ test('callSpread, sell options', async t => {
+    bucksMint,
+    bucks,
+    zoe,
+    amountMaths,
+  } = setup();
+  const installation = await installationPFromSource(zoe, pricedCallSpread);
+
+  // Alice will create a call spread contract, and give the invitations
+  // to Bob and Carol. Bob and Carol will fund and exercise, then promptly
+  // schedule collection of funds. The spread will then mature, and both will
+  // get paid.
+
+  // Setup Bob
+  const bobBucksPurse = bucksIssuer.makeEmptyPurse();
+  const bobBucksPayment = bucksMint.mintPayment(bucks(225));
+  // Setup Carol
+  const carolBucksPurse = bucksIssuer.makeEmptyPurse();
+  const carolBucksPayment = bucksMint.mintPayment(bucks(75));
+
+  const manualTimer = buildManualTimer(console.log, 0);
+  const priceAuthority = await makeTestPriceAuthority(
+    amountMaths,
+    [20, 45, 45, 45, 45, 45, 45],
+    manualTimer,
+  );
+  // underlying is 2 Simoleans, strike range is 30-50 (doubled)
+  const terms = harden({
+    expiration: 3,
+    underlyingAmount: simoleans(2),
+    priceAuthority,
+    strikePrice1: moola(60),
+    strikePrice2: moola(100),
+    settlementAmount: bucks(300),
+    timer: manualTimer,
+  });
+  // Alice creates a pricedCallSpread instance
+  const issuerKeywordRecord = harden({
+    Underlying: simoleanIssuer,
+    Collateral: bucksIssuer,
+    Strike: moolaIssuer,
+  });
+  const { creatorFacet } = await zoe.startInstance(
+    installation,
+    issuerKeywordRecord,
+    terms,
+  );
+  const invitationPair = await E(creatorFacet).makeInvitationPair(75);
+  const { longInvitation, shortInvitation } = invitationPair;
+
+  const invitationIssuer = await E(zoe).getInvitationIssuer();
+  const longAmount = await E(invitationIssuer).getAmountOf(longInvitation);
+  const shortAmount = await E(invitationIssuer).getAmountOf(shortInvitation);
+
+  const longOptionValue = longAmount.value[0];
+  t.is('long', longOptionValue.position);
+  const longOption = longOptionValue.option;
+
+  // Bob makes an offer for the long option
+  const bobProposal = harden({
+    want: { Option: longOption },
+    give: { Collateral: bucks(longOptionValue.collateral) },
+  });
+  const bobPurchaseSeat = await zoe.offer(await longInvitation, bobProposal, {
+    Collateral: bobBucksPayment,
+  });
+  // bob gets an option, and exercises it for the payout
+  const bobOption = await bobPurchaseSeat.getPayout('Option');
+  const bobOptionSeat = await zoe.offer(bobOption);
+
+  const bobPayout = bobOptionSeat.getPayout('Collateral');
+  const bobDeposit = assertPayoutDeposit(
+    t,
+    bobPayout,
+    bobBucksPurse,
+    bucks(225),
+  );
+
+  const shortOptionValue = shortAmount.value[0];
+  t.is('short', shortOptionValue.position);
+  const shortOption = shortOptionValue.option;
+
+  // carol makes an offer for the short option
+  const carolProposal = harden({
+    want: { Option: shortOption },
+    give: { Collateral: bucks(shortOptionValue.collateral) },
+  });
+  const carolPurchaseSeat = await zoe.offer(
```

Comment:
Same here. I wouldn't call this a purchase. It's escrowing margin.

Reply:
ditto